langgraph/src/pregel/index.ts

```diff
@@ -360,13 +398,6 @@ export class Pregel
         checkpoint = await createCheckpoint(checkpoint, channels);
         await this.checkpointer.put(config, checkpoint);
       }
-
-      // interrupt if any channel written to is in interrupt list
-      if (
-        pendingWrites.some(([chan]) => this.interrupt?.some((i) => i === chan))
```

Comment:
If this logic needs to be migrated, I haven't done so yet.

Reply:
it does

Reply:
but can be in next pr